_editorial_policies/01_bylaws.md

```diff
@@ -0,0 +1,149 @@
+
+   1. No Section Lead Editor shall assign an Associate Editor if they
+have a conflict of interest with a paper. No Associate Editor may
+manage a paper if they have a conflict of interest, and shall notify
+the Section Lead Editor if they do have a conflict.
+
+   1. If a Section Lead Editor has a conflict of interest, the
+assignments to Associate Editors will be made by another Section Lead
+Editor.  The order will be cyclic; if Lead Section Editor A has a
+conflict, it will got to Lead Section Editor B, unless they also have
+a conflict, in which case it will go to C, and so on, returning to A
+when the sections are exhausted. In the case that all of the lead
+editors have conflicts, the Managing Editors will jointly pick an
+Associate Editor who will then assign another Associate Editor to
+manage the manuscript.
+      1. Comparison of Molecular Simulation Package Editor
+      1. Tutorials Editor
+      1. Best Practices Editor
+      1. Review Editor
+      1. Lessons Learned Editor
+      1. continuing with editors of other sections, in order they are chronologically added to the journal.
+
+1. Fraud and Plagiarism
+
+    1. Definitions:
+
+       1. 'Plagiarism' is defined as including any clearly recognizable
+amount of text from a previous publication without attribution.
+
+       1. 'Fraud' is defined as any false representation of the circumstances of the paper or the data contained in the paper. It could include fraudulent data, or fraudulent reviews.
+
+    1. If there is a suspicious or accusation of fraud or plagiarism, a
+       committee of three members of the editorial board will be chosen by
+       the managing editors to investigate.  If this committee votes
+       unanimously that fraud or plagiarism occurred, then the paper will
+       be removed from the journal, and the author will not be allowed to
+       submit to LiveCoMS for three years.
+
+1. Review Process
+
+    1. Review criteria that editors use shall be posted on the author
+       instructions section of the website. These review criteria will be
+       revised for clarity periodically in response to feedback, with the
+       approval of the Section Lead Editor.
+
+    2. Section Lead Editors make the final decision about reviews in
+       consultation with the associate editor assigned to the
+       paper. In case of an appeal on the decision, the Section Lead
+       Editor will select another associate editor to review all
+       materials associated with the decision. All authors must agree
+       to appeal the decision. If the second editor is in agreement
+       with the first editor, the decision will stand and cannot be
+       additionally appealed.  If the second editor disagrees with the
+       first decision, a third editor will be selected by the Section
+       Lead Editor, and a majority vote will be taken, which will be
+       final.
+
+1. Amendments
+
+   These bylaws may be amended by two thirds of the vote of the entire
```

Comment:
Do we need to spell out a similar mechanism for their initial adoption?

Reply:
And, in keeping with documents elsewhere, "a two thirds vote of the responding members of the entire editorial board" and should we say given how long notice?
(I just want to avoid the scenario where we want to amend, but 1/3 of our people are too busy to vote so we can't actually amend anything without calling them one at a time and pinning them down. In my view it should be like a presidential election -- you vote when your vote is due in order to get it counted, and what we're interested in is whether 2/3 of those votes are for the amendment.)

Reply:
Added a section on voting earlier.